app/views/layouts/application.html.erb

```diff
@@ -11,5 +11,11 @@
     environment: GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment,
     product_name: "User intent explorer"
   }%>
-<% end %>
 
+  <div class="govuk-width-container">
+    <main class="govuk-main-wrapper " id="main-content" role="main">
```

Comment:
You probably don't need the extra ` ` after `govuk-main-wrapper` (I'm aware my Eligibility Viewer has this problem too), but it's nothing major.

Reply:
👍 good spot thanks - updated in https://github.com/alphagov/govuk-user-intent-survey-explorer/commit/5bb2aa66b4ff697afea9a1a7ef7056147a0846b0